napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1563,8 +1563,11 @@ def get_optics(self):
 
         for port_ts in port_ts_l:
             port = port_re.search(port_ts).group(1)
+            # No transceiver is present in those case
             if "transceiver is not present" in port_ts:
                 continue
+            if "transceiver is not applicable" in port_ts:
```

Comment:
Just a personal curiosity... what does "transceiver is not applicable" mean in this context? :thinking:

Reply:
@mirceaulinic  the *not applicable* state applies to the ports related to the HP C7000 Enclosure B22 switches.